packages/expo-video/plugin/src/withExpoVideo.ts

```diff
@@ -1,4 +1,28 @@
-import { ConfigPlugin, withInfoPlist } from '@expo/config-plugins';
+import { ConfigPlugin, withInfoPlist, withAndroidManifest } from '@expo/config-plugins';
+import { Manifest } from '@expo/config-plugins/build/android';
```

Comment:
```suggestion
import { AndroidConfig, ConfigPlugin, withInfoPlist, withAndroidManifest } from '@expo/config-plugins';
const { Manifest } = AndroidConfig;
```

Don't use the nested imports.